docs/Intro-Start-Here/HT-Updating-The-Firmware.md

```diff
@@ -0,0 +1,105 @@
+that are available for this procedure are documented below in [Other firmware flashing
+tools](#other-firmware-flashing-tools).
+
+After extracting the bundle contents, navigate to the `console/` directory and run the FOME console program (e.g.
+`fome_console.exe` if using Windows).  After first confirming **TunerStudio is not running**, connect the ECU to the
+computer.  FOME console will automatically detect the board and present similar to the display below:
+
+![image](Updating-FW/fomeconsole.png)
+
+Next, click *Update Firmware* and leave the board connected until the display notifies that the update was successful.
+Only once the console has presented the success message, disconnected and re-connect the ECU into the computer to reboot
+it with the new firmware.
+
+If the update does not complete successfully, the firmware may need to be loaded using another mechanism documented
+below in [Other firmware flashing tools](#other-firmware-flashing-tools).
+
+## Setting up TunerStudio
+
+With the ECU connected, open TunerStudio with the associated project.  TunerStudio will recognise that the project's ECU
+definition is out of date and display a prompt to download or supply the new definition.
+
+Next, under the *Controller* tab, open the *Popular Vehicles* menu and click the top button to load the preset for your
+vehicle (**not the button to *Reset firmware settings***).
+
+![image](Updating-FW/popularvehicles.png)
+
+Once the preset has been loaded, you can either start tuning or load your tune from the older firmware versions under
+*File > Load Tune (msq)*. After that, the update is complete!
+
+## Other Firmware Flashing Tools
+
+:::note
+
+Some boards, notably those with processors utilizing certain dual bank memory layout, like the STM32F7, suffer from an
+issue such that flashing does not succeed correctly, due to an error with the erasure mechanism.  For these situations,
+manually erase the flash memory before flashing; i.e. perform a full/mass erase procedure first.
+
+:::
+
+In the event of a firmware update/flash failure, or some other firmware corruption issue, such that FOME console nor
+TunerStudio recognize the ECU when connected, the firmware needs to be loaded to the ECU using another tool.  In these
+situations, the FOME console cannot communicate with the firmware and so is unable to configure it into bootloader (or
+DFU) mode for updating.  To flash the firmware, the ECU must be manually entered into the bootloader/DFU mode.
+
+### STM32CubeProgrammer
+
+STM32CubeProgrammer is an officially STMicroelectronics supported tool to flash STM32 processors.  Information about
+and how to download and install the tool can be found on [the STM
+wiki](https://wiki.st.com/stm32mpu/wiki/STM32CubeProgrammer).
+
+For the ECU, you will need to access buttons or pins on the main board to force the processor into bootloader/DFU mode.
```

Comment:
This whole paragraph could be bumped up a heading, as it applies generally regardless of the tool in use.